projects/1.16/assets/tinkers-archery/tinkersarchery/book/awesome_archery/zh_cn/modifier_arrow/flares.json

```
@@ -0,0 +1,6 @@
+{
+  "title": "闪光强化",
+  "text": [
+    { "text": "闪光为箭矢添加了更多视觉效果，但不会以任何方式影响其工作方式。" }
```

```suggestion
  "title": "火花强化",
  "text": [
    { "text": "火花为箭矢添加了更多视觉效果，但不会以任何方式影响其工作方式。" }
```
与上方修改联动，两边需要一起更改或一起拒绝